Assets/Scripts/TankService.cs

```diff
@@ -0,0 +1,28 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
```

Comment:
Service class should be a singleton class for one MVC its work is to communicate between other MVC's.

Reply:
Service class should be inherited from MonoSingleton class.